ADD APIs related to personal todo

sukyeongs opened the pull request:
1) `GET todos/my `: 현재 날짜 개인투두 조회 기능
2) `POST todos/my`: 현재 날짜 개인투두 추가 기능
3) `GET todos/my/<str:date>`: 특정 날짜 개인투두 조회 기능
4) `POST todos/my/<str:date>`: 특정 날짜 개인투두 추가 기능
5) `POST todos/my/<int:id>/check`: 개인투두 완료 기능
6) `POST todos/my/<int:id>/delete`: 개인투두 삭제 기능
7) `POST todos/my/<int:id>/edit`: 개인투두 수정 기능
8) `POST todos/my/<int:id>/delay`: 개인투두 미루기 기능

Commit:
feat(todo/model): add User_personal_todo model

add model and migrations

Commit:
feat(todo/serializers.py): add UserPersonalTodoSerializer

Commit:
feat(todo/urls.py): add urls and change url to plural

혹시 몰라서 url 복수형으로 바꿨습니다!

Commit:
refactor, style(todo/views.py): create method to refactor and edit tabs

코드를 더 간결하게 하기 위해서 (리팩토링 하고자) 자주 쓰는 함수를 따로 빼고 주석 탭 간격 수정했습니다!

Commit:
feat(todo/views.py): add views related to personal todo

1. 현재 날짜의 개인 투두 조회, 추가
2. 특정 날짜의 개인 투두 조회, 추가
3. 개인 투두 완료
4. 개인 투두 삭제
5. 개인 투두 수정
6. 개인 투두 미루기 
기능 view들 작성 완료했습니다!

Commit:
chore(todo/urls.py): add words

주석에 '추가' 추가

ssssujini99 commented:
개인투두삭제기능, 개인투두수정, 개인투두내일하기에서는 user_personal_todo에서 객체 가져올 때 어차피 pk값으로 가져오기때문에 finish_flag = False가 굳이 필요 없을것같아!
음 삭제나 수정은 완료된 투두를 삭제나 수정할수도 있는거니까 finish_flag = False는 없는게 더 좋을것같앙!!

그리구 현재 날짜 조회,추가 이거랑 특정 날짜 조회,추가 이거는 그냥 날짜 받아오는걸로 아예 하나로만 만들어두 될것같은뎅?? 똑같은 View가 두개 있는거 같아성 ㅎㅎ

그리구 개인투두삭제에서는 우리 delete_flag대신 그냥 이거 객체 없애면되는거아니야? 음 그냥 delete하면 될것같아서!!

히히 내 피드백은 요기까지! 너가 보고 필요한것만 수정하면 될것같아🥰 넘 수고했옹

sukyeongs commented:
핫슁 피드백 부분 반영할 거 반영하고! 기획이나 디자인분들께 완료된 투두는 수정할 수 있는지 없는지 물어봐야겄슴다!! 좋은 피드백 넘넘 감사두립니댜😍

Commit:
chore(todo/model): delete delete_flag field

Commit:
chore(todo/views, urls.py): merge urls into one and delete get_today method

default 값인 현재 날짜의 투두를 불러오는 url도 url에 date가 들어가도록 수정(오늘 날짜를 가져오는건 프론트쪽에서 해줄 수 있다고 함!)

Commit:
chore(PersonalTodoDeleteAPIView): delete delete_flag

투두가 삭제되면, delete 메소드를 사용하여 객체 자체를 지워버리도록 수정 (delete_flag field 없앰)

Commit:
chore(todo/views.py): delete finish_flag in filtering

완료된 투두도 수정/삭제/미루기가 가능하게 수정
